src/main/java/frc/robot/VisionFieldLayoutRecognizer.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot;
+
+import java.util.ArrayList;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.AutonomousPath.VisionPathName;
+
+
+/*
+ * VisionFieldLayoutRecognizer
+ * 
+ * Version 1.0
+ *
+ * 02/27/2021
+ * 
+ * EnTech 
+ */
+
+ public class VisionFieldLayoutRecognizer
+    implements PixyFieldPoseResolver{
+
+     
+    
+    public VisionFieldLayoutRecognizer() {
+      
+    }
+
+     public AutonomousPath detectPose(ArrayList<FieldVisionInput> arraylistFieldVisionInput) {
+        
+      AutonomousPath objectAutonomousPath;
```

Comment:
dont use default ( package protected scope unless you have a good reason and can explain it )